docs/subnets/how-to-upgrade.md

```diff
@@ -0,0 +1,469 @@
+Please consult 
+https://docs.avax.network/subnets/customize-a-subnet#network-upgrades-enabledisable-precompiles 
+for more information
+Use the arrow keys to navigate: ↓ ↑ → ← 
+? Press [Enter] to continue, or abort by choosing 'no': 
+  ▸ Yes
+    No
+```
+
+Go ahead and select `Yes` if you understood everything and you agree.
+
+You see a last note, before the actual configuration wizard starts:
+
+```shell
+Avalanchego and this tool support configuring multiple precompiles. 
+However, we suggest to only configure one per upgrade.
+
+Use the arrow keys to navigate: ↓ ↑ → ← 
+? Select the precompile to configure: 
+  ▸ Contract Deployment Allow List
+    Manage Fee Settings
+    Native Minting
+    Transaction Allow List
+```
+
+Refer to [Precompiles](./customize-a-subnet#precompiles) 
+for a description of available precompiles and how to configure them. 
+
+Make sure you understand Precompiles thoroughly and how to configure them before 
+attempting to continue.
+
+For every precompile in the list, the wizard guides you to provide correct
+information, prompting relevant questions. For the sake of this tutorial, select
+`Transaction Allow List`. [Restricting Who Can Submit
+Transactions](./customize-a-subnet#restricting-who-can-submit-transactions)
+describes what this precompile is about.
+
+```shell
+✔ Transaction Allow List 
+Set parameters for the "Manage Fee Settings" precompile
+Use the arrow keys to navigate: ↓ ↑ → ← 
+? When should the precompile be activated?: 
+  ▸ In 5 minutes
+    In 1 day
+    In 1 week
+    In 2 weeks
+    Custom
+```
+
+This is actually common to all precompiles: they require an 
+activation timestamp.
+If you think about it, it makes sense: you want a synchronized activation of your precompile.
+So think for a moment about to what you want to set the activation timestamp to.
+You can select one of the suggested times in the future, or you can select a custom one.\
+Select `Custom` here, and see the format required:
+
+```shell
+✔ Custom
+✗ Enter the block activation UTC datetime in 'YYYY-MM-DD HH:MM:SS' format: 
+```
+
+The format is `YYYY-MM-DD HH:MM:SS`, therefore `2023-03-31 14:00:00` would be a valid timestamp.
+Notice the `✗` at the beginning of the line. The CLI tool does input validation, so if you 
+provide a valid timestamp, the `x` disappears:
+
+```shell
+✔ Enter the block activation UTC datetime in 'YYYY-MM-DD HH:MM:SS' format: 2023-03-31 14:00:00
+```
+
+The timestamp must be in the **future**, so make sure you use such a 
+timestamp should you be running this tutorial after `2023-03-31 14:00:00` ;-) .
+It's your **local** timestamp, by the way.
+
+After you provided the valid timestamp, proceed with the precompile specific configurations:
+
+```shell
+The chosen block activation time is 2023-03-31 14:00:00
+Use the arrow keys to navigate: ↓ ↑ → ← 
+? Add 'adminAddresses'?: 
+  ▸ Yes
+    No
+```
+
+The addresses added in this section allow to add other admins and/or add enabled addresses 
```

Comment:
```suggestion
The addresses added in this section will be enabled to add other admins and/or add enabled addresses 
```

Reply:
This will also result in passive mode warnings

Reply:
How about changing to the following?
```
This will enable the addresses added in this section to add other admins and/or add enabled addresses 
```

Reply:
ok changed accordingly